eng/tools/tsp-client-tests/tsconfig.json

```diff
@@ -1,6 +1,12 @@
 {
   "extends": "../tsconfig.json",
   "compilerOptions": {
-    "outDir": "./dist",
-  }
+    "composite": true,
+    "outDir": "dist",
+    "rootDir": "."
+  },
+  "include": [
+    "src/**/*.ts",
+    "test/**/*.ts"
+  ]
```

Comment:
All sub-projects should include these configs by default.  They cannot be set in the base config, because paths are relative to tsconfig.json.